packages/monorepo-scripts/src/copy_contract_artifacts.ts

```diff
@@ -0,0 +1,41 @@
+const pkgNames = process.argv.slice(2);
+import * as artifacts from '@0x/contract-artifacts';
+import * as fs from 'fs';
+import * as path from 'path';
+/*
+ * @param dir        the directory name under 0x-monorepo/contracts, e.g. 'exchange', 'extensions'
+ * @param artifact   the artifact name without extension, e.g. DutchAuction, Exchange
+ */
+function shouldSkip(dir: string, artifact: string): boolean {
+    return (dir === 'extensions' && artifact === 'Exchange') || (dir === 'extensions' && artifact === 'WETH9');
+}
+const artifactsToPublish = Object.keys(artifacts);
+
+const contractsPath = path.join(__dirname, '../../../contracts');
+const contractsDirs = pkgNames.map(pkg => pkg.split('/contracts-')[1]);
```

Comment:
There is an implicit assumption that all packages under the `contracts` dir will always start with `contracts-`. I think we should make this assumption explicit and throw a helpful error message if we find this not to be the case.